index.html

```diff
@@ -126,7 +126,58 @@
 				</p>
 			</div> <!-- cd-faq-content -->
 		</li>
-	</ul> <!-- cd-faq-group -->
+	</ul> 
+        <ul id="frameworki" class="cd-faq-group">
+                <li class="cd-faq-title"><h2>Frameworki</h2></li>
+                <li>
+                        <a class="cd-faq-trigger" href="#0">Dlaczego hejtujemy Laravela?</a>
```

Comment:
W zasadzie to nie hejtujemy bo nie można nazwać hejtem powiedzenia, że białe jest białe a czarne jest czarne.